test/fixtures/test-runner/output/lcov_reporter.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+require('../../../common');
+const fixtures = require('../../../common/fixtures');
+const spawn = require('node:child_process').spawn;
+
+const child = spawn(process.execPath,
+                    ['--no-warnings', '--experimental-test-coverage', '--test-reporter', 'lcov', fixtures.path('test-runner/output/output.js')],
+                    { stdio: 'pipe' });
+// eslint-disable-next-line no-control-regex
+child.stdout.on('data', (d) => process.stdout.write(d.toString().replace(/[^\x00-\x7F]/g, '').replace(/\u001b\[\d+m/g, '')));
```

Comment:
What does this line do? Can you add a comment?

Reply:
Ah, I'd copied this from the spec_reporter.js, but I guess that was replacing colour control characters. It's unnecessary for plain text output like this, so I've removed it and spawn will inherit stdio.